events/wc_events/wc_central_kalimdor_events.txt

```diff
@@ -0,0 +1,1614 @@
+﻿namespace = tauren
+namespace = cen_kal
```

Comment:
I have some Earth Mother faith specific events and other Central Kalimdor specific ones, that's why I opted on two namespaces. With that in mind, you still want me to combine the namespaces, or should I leave them as is?

Reply:
it is related events through one on_actions and one PR, in vanilla and in main mod people always put it in one file/namespace.

Reply:
Is it possible to do this change to the event names only, while the localization can maintain the old names. It will be hard to regulate all of the different names and descriptions for the text part